src/sentry/data_export/processors/discover.py

```diff
@@ -0,0 +1,81 @@
+from __future__ import absolute_import
```

Comment:
This file is basically all the actual copied discover code, it's all handled in the constructor, then in the task, we run the `data_fn`.